### PR TITLE
Update CONTRIBUTING.MD

### PR DESCRIPTION
**Pull request by asturur**
- clarify import orders and unnecessary changes
- ~~add some help with tests~~ redo testing section
- clarify around refactors / current code structure
- clarify around timings

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -95,13 +95,14 @@ Fabric is an open source project 🦄 and as such depends on the **genuine effor
 
 ### ✅ Guidelines
 
+- **Be patient** \
+  Sometimes it takes time to get back to you. Someone eventually will come back at you. Having a small and concise and super clear change will make maintainers more prone to handle it quickly.
```
Comment (ShaMan123) [inline CONTRIBUTING.md:99]:
```suggestion
  Sometimes it takes time to get back to you. Someone eventually will. Having a small, concise and super clear change will make maintainers more prone to handle it quickly.
```

**Review by ShaMan123 — CHANGES_REQUESTED** [review]
Some changes, typos and questions

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -95,13 +95,14 @@ Fabric is an open source project 🦄 and as such depends on the **genuine effor
...
 - **Code Style** \
   Fabric uses [`prettier`][prettier] to format files and [`eslint`][eslint] for linting (`npm run lint -- --fix`).\
   To enjoy a seamless dev experience add the [`Prettier - Code formatter`][prettier_extension] extension via the extensions toolbar in VSCode.
-- **⛔ `dist`** \
-  Commit changes to [source files](src). Don't commit the generated [distribution files](dist).
+  Do not reorder imports. Unrelevant changes in pr that are not created by the basic prettier aren't needed nor welcome.
 - **Tests** \
-  PRs must be backed with relevant tests, follow [TESTING](#-testing).
+  PRs must be backed with relevant tests, follow [TESTING](#-testing). If you are a first timer with tests, asks for help.
```
[the remark this answers is not in the document]
Comment (ShaMan123) [reply CONTRIBUTING.md:105]: ### Test Your Changes

- start dev mode (watch for changes)
`npm run build -- -f -w`
- check existing tests pass (on chrome, can be a different platform)
`npm test -- -c chrome -a`
- handle failing tests
`npm test -- -c chrome`
   - fix logic
   - rerun tests by selecting the files with the failing tests (to save time)
image
    
- add tests that cover your change (aiming to 100% of cases)
  - unit test example
  - visual test example
- visit the PR on github to see all checks have passed (different platforms and config are covered by the checks)

**Comment by ShaMan123** [comment]
redid the testing section

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -110,6 +111,7 @@ Fabric is an open source project 🦄 and as such depends on the **genuine effor
 - **1️⃣ PR per feature/bug** \
   Create a new branch for every pull request.\
   If you want to do more than one thing, create multiple pull requests 💪.
+  If your bug fix or feature requires a refactor, don't refactor. Commit the bugfix or the feature with the current code structure, let it sink, give some time to surface issues with the change, then when the bug or the feature seem solid, a refactor or code improvement can be tried
```
[the remark this answers is not in the document]
Comment (ShaMan123) [reply CONTRIBUTING.md:114]: Maybe be even add a section about discussing motivation etc.

[the remark this answers is not in the document]
Comment (asturur) [reply CONTRIBUTING.md:114]: you can always fix a bug, even if is with shitty code.
The purpose of the message is to not have maintainers have to evaluate a bug or a feature and a refactor all together.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -91,17 +91,18 @@ Fabric is an open source project 🦄 and as such depends on the **genuine effor
...
+  Sometimes it takes time to get back to you. Someone eventually will. Having a small, concise and super clear change will make maintainers more prone to handle it quickly.
 - **Code Style** \
   Fabric uses [`prettier`][prettier] to format files and [`eslint`][eslint] for linting (`npm run lint -- --fix`).\
   To enjoy a seamless dev experience add the [`Prettier - Code formatter`][prettier_extension] extension via the extensions toolbar in VSCode.
-- **⛔ `dist`** \
-  Commit changes to [source files](src). Don't commit the generated [distribution files](dist).
+  Do not reorder imports. Irrelevant changes in a PR that are not created by prettier aren't needed nor welcome.
 - **Tests** \
-  PRs must be backed with relevant tests, follow [TESTING](#-testing).
+  PRs must be backed with relevant tests, follow [TESTING](#-testing). We are here to help.
```
Comment (asturur) [inline CONTRIBUTING.md:105]: There are developer that do not do tests. I want to be clear, if you never wrote a test, ask for help. We are here to help is so generic.
Someone may not know the scope of a unit test, a rendering test or a more convolute integration test

Comment (ShaMan123) [reply CONTRIBUTING.md:105]: OK

**Pull request merged**